### DISCUSSION TITLE
Account Suspended

### DISCUSSION THREAD

**Discussion by monster-zero-sugar** [community/community#122266] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, 

I understand that the community can't get my account back. I just want to collect info to figure out what's going on.

My account, `n1md7`, got suspended out of nowhere. I can't access it anymore, and it's like it never existed. I had over a hundred repositories, and now they're all gone.

The GitHub login message says I violated their policy but doesn't explain what specifically. I've created a support ticket, but it's been a week, and I haven't heard back. All my life's work is there 😞 

Before the suspension, I noticed a weird repository that appeared by "itself". It had a random(UUID) name and talked about hacking. I deleted it and checked my security settings, but I couldn't find any clues about who made it. As a precaution, I disabled some Github auth apps.

I don't think someone hacked my account directly because I didn't get any alerts about unusual activity like email or phone notifications (every new device login requires a 2FA code). Even if they did, there should've been some trace of it. Could it be a problem with my access token (PAT leaked somehow) or a hidden bug that someone found?

I'd appreciate your thoughts on this. How could someone mess with my account and delete everything without setting off any alarms, even though I have Two-Factor Authentication (2FA) turned on?

**Comment by angry-mustard-it** [comment]
I call it Git Gate. It is a phenomena of account sellings, scams, and even malicious account hackings.

**Reply by monster-zero-sugar** [reply]
Is there a definitive way to avoid it?

**Reply by angry-mustard-it** [reply]
Apparently, you need to stay away from scam websites that get access to your account, unlike YouTube. Also avoid dangerous download links, account accessors and stay away from bots that took over your account. You need a incredibly strong password in order to do this so you dont leak your password so there are grabbers who try to scam and do mean-spirited behavior.

**Reply by monster-zero-sugar** [reply]
Thank you for your response. 

> You possibly visited suspicious websites so remove them from the browsing history. Use an antivirus, and try to use an adblocker, for example Ublock Origin

I believe this is pretty much common sense in general to avoid getting hacked.

I use a password generator for my accounts, so I'm pretty confident my old or other passwords haven't been used to gain access to my account even if they were leaked anywhere. I browse with Brave, which has a built-in ad blocker, and I always have [Privacy Badger](https://privacybadger.org/) installed. I doubt my web browsing has anything to do with this issue but you never know 🤔

**Comment by angry-mustard-it** [comment]
> Is there a definitive way to avoid it?

You possibly visited suspicious websites so remove them from the browsing history. Use an antivirus, and try to use an adblocker, for example Ublock Origin

**Comment by salma-elbakkouri** [comment]
I'm sorry to hear about your GitHub account trouble. It sounds frustrating. It's possible your account was compromised through a PAT leak, or a hidden bug because even with 2FA, vulnerabilities are still there. Keep following up with GitHub support, I hope they can provide a solution

**Comment by monster-zero-sugar** [comment]
Yeah, I don't see any other ways but to wait for the GitHub support
response.

Now I almost certain it was my PAT that got leaked since I had generated
around 10-15 for my different use cases with no expiration date specified.

**Comment by monster-zero-sugar** [comment]
I'm posting the follow-up for this issue. 

I've regained access to my account. It was suspended due to security measures triggered by malicious activity. The root cause was indeed the leakage of my Personal Access Token (PAT). A malicious actor exploited it to generate issues across various posts using an automated script.

This is what support texted me:
> During the account compromise, it appears the bad actor created multiple GitHub Issues from your account used to incentivize engagement with cryptocurrency airdrops, an activity prohibited by our [Acceptable Use Policies](https://docs.github.com/site-policy/acceptable-use-policies/github-acceptable-use-policies).
 
> Unfortunately, we do not have the capacity for the time being to delete these issues for you. We would kindly request that you edit them yourself whenever possible. If you are familiar with the GitHub REST API, we would suggest [updating all the issues at once](https://docs.github.com/rest/issues/issues?apiVersion=2022-11-28#update-an-issue--parameters) to remove the malicious content using the API if that can help.

I'm intrigued by the REST API but couldn't discover a method to retrieve a list of all issues recently created by my user alone. It seems to demand both the repository name and the project owner for this query. Although I came across the endpoint `https://api.github.com/issues`, it solely displays issues assigned to my user, not those I've created.

Do you have any suggestions on how I might list the issues created by my user without needing the repository name?

I just noticed that the account is usable but it has some issues. I'm marked as spam and cannot use GitHub login anymore

<img width="590" alt="image" src="https://github.com/community/community/assets/129964421/7fd2fda8-554f-4436-a9c9-5098463916d3">

Also, my GitHub pages no longer work. There is no error in the pipeline build or anywhere when I push changes. It just acts like there are no GitHub pages enabled at all while I can confirm that it is enabled. Do you think it is related to my account being marked as spam?

<img width="588" alt="image" src="https://github.com/community/community/assets/129964421/4f924e7f-036a-443b-89b7-a90b648c39b1">

I asked GH support these questions, but it took 2 weeks before and I have no expectation to get a reply before that, so looking for possible answers here if I can get any.

**Comment by ghost** [comment]
@monster-zero-sugar 
Did you get a response? Mine got suspended too, been more than a month and still no response.
ticket: #2719793

**Reply by monster-zero-sugar** [reply]
After the first interaction, I did not get any response. 

It is so sad they have not improved support and take ages to respond.

I have access to my account but it is not visible to anybody else. I am marked as SPAM and it does not allow me to do anything other than viewing my code.

And they still charge me for having an account without fixing it. :|
